### DISCUSSION TITLE
Can PubPub use multiple custom domains?

### DISCUSSION THREAD

**Discussion by zblace** [knowledgefutures/pubpub#1714] [Support] [answered]
I am thinking to compile content of more than one project and brand together and I wonder if it is an option for PubPub service to use multiple custom domains?

**Comment by gabestein** [comment, marked as answer]
Hello, thanks for considering PubPub. Yes, you can have one domain per community, and they can be subdomains, as well as top-level domains, so for example you could have community1.yourdomain.org, community2.yourdomain.org, etc. 

You could then link them together from a single overarching community. Here's one such example: https://mitpressonpubpub.mitpress.mit.edu/
